packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/Test.test.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
```

Comment:
This file should be renamed to depict its purpose

Reply:
This is temporary to conclude some confusions.

There is a on going offline discussion on the behavior of changing timezone for a date time picker. 
Sure will be deleted once we have a final answer.